.travis/install.sh

```diff
@@ -11,29 +11,25 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     eval "$(pyenv init -)"
 
     case "${TOXENV}" in
-        py26)
-            pyenv install 2.6.9
-            pyenv global 2.6.9
-            ;;
         py27)
             curl -O https://bootstrap.pypa.io/get-pip.py
             python get-pip.py --user
             ;;
-        py33)
-            pyenv install 3.3.6
-            pyenv global 3.3.6
-            ;;
-        py34)
-            pyenv install 3.4.5
-            pyenv global 3.4.5
-            ;;
         py35)
-            pyenv install 3.5.2
-            pyenv global 3.5.2
+            pyenv install 3.5.9
```

Comment:
This change is nice but not necessary for Travis v3 API.

Reply:
okay, reverted to old file.